### PR TITLE
Update initVar.sqf

### PR DESCRIPTION
**Pull request by AlexTriada**
I made this for myself, looks better and optimisation also

### CODE REVIEW

**Comment by Spoffy** [comment]
Just working through this. Looks good for the most part, though I'm not sure '[a, b] select (true)' is more readable than the if statements - you have to know a decent amount about how 'select' works to make sense of it, whereas 'if x then a else b' is rather clear.

Most of the places you use 'apply' are improvements to readability, particularly where you condensed the magazine code. Worth noting that 'apply' is slightly slower than 'forEach' though.
```
Result:
1.6 ms

Cycles:
625/10000

Code:
testArray apply { _x + 1}
```

```
Result:
1.2987 ms

Cycles:
770/10000

Code:
{ _x + 1} forEach testArray
```

File: A3-Antistasi/initVar.sqf [hunk 1]
```diff
@@ -132,362 +124,724 @@ hasFFAA = false;
...
+};
+
+diag_log format ["%1: [Antistasi] | INFO | initVar | Assigning vehicle Types", servertime];
+
+vehNormal = vehNATONormal + vehCSATNormal + [vehFIATruck, vehSDKTruck, vehSDKLightArmed, vehSDKBike, vehSDKRepair];
+vehBoats = [vehNATOBoat, vehCSATBoat, vehSDKBoat];
 vehAttack = vehNATOAttack + vehCSATAttack;
 vehPlanes = vehNATOAir + vehCSATAir + [vehSDKPlane];
 vehAttackHelis = vehCSATAttackHelis + vehNATOAttackHelis;
-vehFixedWing = [vehNATOPlane,vehNATOPlaneAA,vehCSATPlane,vehCSATPlaneAA,vehSDKPlane] + vehNATOTransportPlanes + vehCSATTransportPlanes;
```
Comment (Spoffy) [inline A3-Antistasi/initVar.sqf]: This is out of date - there's a subtle difference here.

Comment (Spoffy) [reply A3-Antistasi/initVar.sqf]: Fixed it.

**Comment by AlexTriada** [comment]
but for some reason I thought that apply was faster, it seems that I was wrong)

**Pull request closed without merging**